services/ipqsservicesample/ui/index.php

```diff
@@ -0,0 +1,257 @@
+        const buttonExecuteApiScoringAddress = document.querySelector('#button-execute-api-scoring-address');
+        
+        const inputApiKeyToken = document.querySelector('#input-api-key-token');
+        
+        const inputEmail = document.querySelector('#input-api-scoring-email');
+        const inputPhone = document.querySelector('#input-api-scoring-phone');
+        const inputUrl = document.querySelector('#input-api-scoring-url');
+        const inputIp = document.querySelector('#input-api-scoring-ip');
+        const inputAddress = document.querySelector('#input-api-scoring-address');
+        
+        const textareaApiScoringResult = document.querySelector('#textarea-api-scoring-result');
+        
+        const apiEndpoints = {
+          KEY: 'http://localhost:8081/ipqs/api/key',
+          EMAIL: 'http://localhost:8081/ipqs/score/email/',
+          PHONE: 'http://localhost:8081/ipqs/score/phone/',
+          URL: 'http://localhost:8081/ipqs/score/url/',
+          IP: 'http://localhost:8081/ipqs/score/ip/',
+        };
+        
+        let apiKeyToken = undefined;
+        
+        buttonSaveApiKeyToken.onclick = function() {
+          let apiKeyTokenValue = inputApiKeyToken.value;
+          
+          if (/^\s*$/.test(apiKeyTokenValue)) {
+            alert('Please provide a valid API Key Token');
+            inputApiKeyToken.value = '';
+            return;
+          }
+          
+          $.ajax({
+            type: 'PUT',
+            url: apiEndpoints.KEY,
+            contentType: 'application/json',
+            data: JSON.stringify({key: apiKeyTokenValue}),
+          }).done(function (response, status) {
+            textareaApiScoringResult.value = JSON.stringify(response);
+          })
+          
+          apiKeyToken = apiKeyTokenValue;
+          inputApiKeyToken.value = '';
+          inputApiKeyToken.placeholder = 'API Key Token is Set (Click Show to Display It)';
+        }
+        
+        buttonClearApiKeyToken.onclick = function() {
+          $.ajax({
+            type: 'DELETE',
+            url: apiEndpoints.KEY,
+          }).done(function (response, status) {
+            textareaApiScoringResult.value = JSON.stringify(response);
+          })
+          
+          apiKeyToken = undefined;
+          inputApiKeyToken.value = '';
+          inputApiKeyToken.placeholder = 'Enter API Key Token';
+        }
+        
+        buttonShowApiKeyToken.onclick = function() {          
+          $.get(apiEndpoints.KEY, function(response, status) {
+            textareaApiScoringResult.value = JSON.stringify(response);
+          })
+        }
+        
+        buttonExecuteApiScoringEmail.onclick = function() {
+          let email = inputEmail.value;
+          let emailEndpoint = apiEndpoints.EMAIL + `${email}`;
+          
+          $.get(emailEndpoint, function(response, status) {
+            textareaApiScoringResult.value = JSON.stringify(response);
```

Comment:
It might be cool to add in some table-ifying for response attributes so things are more readable, or we can consider doing a pretty-print. This isn't crucial, but it'd be be nice to see.